cli/program_state.rs

```diff
@@ -65,11 +68,50 @@ impl ProgramState {
     let dir = deno_dir::DenoDir::new(custom_root)?;
     let deps_cache_location = dir.root.join("deps");
     let http_cache = http_cache::HttpCache::new(&deps_cache_location);
+
+    let default_ca_stores = vec!["mozilla".to_string()];
+    let supported_ca_stores = vec!["mozilla".to_string(), "system".to_string()];
+
+    let mut root_store = RootCertStore::empty();
+    let ca_stores: Vec<String> = flags
+      .ca_stores
+      .clone()
+      .or_else(|| {
+        let env_ca_store = env::var("DENO_TLS_CA_STORE");
+        match env_ca_store {
+          Ok(arg) => Some(arg.split(',').map(|s| s.to_string()).collect()),
+          _ => None,
+        }
+      })
+      .or_else(|| Some(default_ca_stores.clone()))
+      .unwrap()
+      .iter()
+      .filter(|store| supported_ca_stores.contains(&store))
```

Comment:
For that matter: unsupported should probably be a fatal error?

Reply:
Ok, this seemed more forgiving but its probably wise to throw an error incase someone misspells "mozila" or something.

Reply:
Ok, it returns an error now if you pass in an unsupported store:

<img width="599" alt="Screen Shot 2021-08-03 at 2 27 20 PM" src="https://user-images.githubusercontent.com/10974/128074439-76e12a11-525a-4541-90f3-3ccc027a3f4e.png">